cmd/pbackend/statuser.go

```diff
@@ -292,19 +294,19 @@ func statuser() {
 		sigint := make(chan os.Signal, 1)
 		signal.Notify(sigint, syscall.SIGINT, syscall.SIGTERM)
 		<-sigint
-		if err := metricsServer.Shutdown(context.Background()); err != nil {
-			logger.Warn().Err(err).Msg("Metrics service shutdown error")
+		if shutdownErr := metricsServer.Shutdown(context.Background()); shutdownErr != nil {
+			logger.Warn().Err(shutdownErr).Msg("Metrics service shutdown error")
 		}
 		close(signalNotify)
 	}()
 
 	go func() {
-		if err := metricsServer.ListenAndServe(); err != nil {
+		if listenErr := metricsServer.ListenAndServe(); listenErr != nil {
 			var errInUse syscall.Errno
-			if errors.As(err, &errInUse) && errInUse == syscall.EADDRINUSE {
-				logger.Warn().Err(err).Msg("Not starting metrics service, port already in use")
-			} else if !errors.Is(err, http.ErrServerClosed) {
-				logger.Warn().Err(err).Msg("Metrics service listen error")
+			if errors.As(listenErr, &errInUse) && errInUse == syscall.EADDRINUSE {
+				logger.Warn().Err(listenErr).Msg("Not starting metrics service, port already in use")
+			} else if !errors.Is(listenErr, http.ErrServerClosed) {
+				logger.Warn().Err(listenErr).Msg("Metrics service listen error")
 			}
 		}
 	}()
```

Comment:
Linter wanted me to change because of variable shadowing, probably a new linter enabled after the upgrade.

This is unrelated rename of `err` variables.